lib/fluent/plugin/in_monitor_agent.rb

```diff
@@ -330,7 +282,7 @@ def all_plugins
     # from the plugin `pe` recursively
     def self.collect_children(pe, array=[])
       array << pe
-      if pe.is_a?(MultiOutput) && pe.respond_to?(:outputs)
```

Comment:
Oh, I see....
I've added additional commit to resolve your commented problem: https://github.com/fluent/fluentd/pull/1151/commits/03cf62e8d1b15ca90a52b979b2d08ec4d06dd6aa